packages/langium/test/grammar/formatting.test.ts

```diff
@@ -0,0 +1,121 @@
+/******************************************************************************
+ * Copyright 2023 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import { beforeAll, describe, test, expect } from 'vitest';
+import { createLangiumGrammarServices, EmptyFileSystem } from 'langium';
+import { expectFormatting, expectFunction } from 'langium/test';
```

Comment:
For some reason does this style of importing break the `expectFunction` behavior. The following suggestion should fix it though.

```suggestion
import { createLangiumGrammarServices, EmptyFileSystem } from '../../src';
import { expectFormatting } from '../../src/test';
```

Reply:
The import of `expectFunction` is needed to override how `expectFormatting` behaves below. Without it the tests fail on my end. This is interesting if you're seeing the exact opposite however, as there may be something up with the implementation based on platform then.

Reply:
Also going with `../../src/test` causes the `expectFormatting` usage to break on my end.

Reply:
Nvm, got it, helps rectify the issue with `expectFunction` needing to used in the first place